docs/_includes/footer.html

```diff
@@ -1,14 +1,18 @@
-<footer class="site-footer">
-  <div class="footer-content">
-    <div class="footer-logo-cont"><img class="footer-logo" src={{ "/assets/images/um-logo-m216.png" | relative_url }} alt="UM logo" /></div>
-    <div class="footer-text">
-      <h2>Information and Technology Services</h2>
-      <h3>How can we help you?</h3>
-      <p>Contact the <a href="https://its.umich.edu/help">ITS Service Center</a>:</p>
-      <ul>
-        <li>Chat: <a style="color: #fff" href="https://chatsupport.it.umich.edu/">chatsupport.it.umich.edu</a></li>
-        <li>Call: <a style="color: #fff" href="tel:734-764-4357">734-764-HELP(764-4357)</a></li>
-      </ul>
+<footer class="c-footer-style mt-5">
+  <div class="container">
+    <div class="row">
+      <div class="col-sm">
+        <h5><img src={{ "/assets/images/um-logo.png" | relative_url }} alt="UM logo" class="img-fluid mb-1 logo-config"></h5>
+        <small class="copy-rights cursor-pointer">&#9400; 2022 The Regents of the University of Michigan</small>
+      </div>
+      <div class="col-sm">
+        <ul class="list-unstyled">
+          <li class="mt-md-3 mt-4 footer-heading-row">ITS Teaching and Learning</li>
+          <li>Website: <a style="color: #fff" href="https://its.umich.edu/help" target="_blank">Contact ITS</a></li>
+          <li>Chat: <a style="color: #fff" href="https://chatsupport.it.umich.edu/" target="_blank">chatsupport.it.umich.edu</a></li>
+          <li>Call: <a style="color: #fff" href="tel:734-764-4357">734-764-HELP(764-4357)</a></li>
```

Comment:
This is probably okay to repeat. You could use `class="link-light"` if you want to make more use of Bootstrap. https://getbootstrap.com/docs/5.1/helpers/colored-links/